### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by jbellmann**

### CODE REVIEW

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<factorypath>
```
Comment (hjacobs) [inline .factorypath:1]: hmm, we don't need that file, do we?

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 # zmon-notification-service
+
+[![Build Status](https://travis-ci.org/zalando/zmon-notification-service.svg?branch=moreTests)](https://travis-ci.org/zalando/zmon-notification-service)
```
Comment (hjacobs) [inline README.md]: I guess the README should reference the master branch :smirk:

File: pom.xml [hunk 3]
```diff
@@ -2,7 +2,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-
+    <parent>
```
Comment (hjacobs) [inline pom.xml:5]: hmm, do we really need that dependency to STUPS parent?

File: README.md (hunk 2 above)
Comment (jbellmann) [inline README.md]: Sure, when it is in master.

File: pom.xml (hunk 3 above)
Comment (jbellmann) [inline pom.xml:5]: Don't you like STUPS?

Comment (jbellmann) [inline pom.xml:5]: I don't want to setup common things (eg. release-plugin) in every project.

**Pull request closed without merging**